buildkite.sh

```diff
@@ -0,0 +1,343 @@
+        echo '      VIBED_DRIVER=vibe-core PARTS=examples ./travis-ci.sh'
+        ;;
+    vibe-d/vibe.d+vibe-core-tests)
+        remove_spurious_vibed_tests
+        echo '      VIBED_DRIVER=vibe-core PARTS=tests ./travis-ci.sh'
+        ;;
+    vibe-d/vibe.d+libasync-base)
+        echo '      VIBED_DRIVER=libasync PARTS=builds,unittests ./travis-ci.sh'
+        ;;
+
+    vibe-d/vibe-core+epoll)
+        echo '      rm tests/issue-58-task-already-scheduled.d # https://github.com/vibe-d/vibe-core/issues/84'
+        echo '      CONFIG=epoll ./travis-ci.sh'
+        ;;
+
+    vibe-d/vibe-core+select)
+        echo '      rm tests/issue-58-task-already-scheduled.d # https://github.com/vibe-d/vibe-core/issues/84'
+        echo '      CONFIG=select ./travis-ci.sh'
+        ;;
+
+    rejectedsoftware/diet-ng)
+        echo "      sed -i '/mkdir build && cd build/,//d' .travis.yml" # strip meson tests
+        use_travis_test_script
+        ;;
+
+    dlang/dub)
+        cat << EOF
+          rm test/issue895-local-configuration.sh # FIXME
+          rm test/issue884-init-defer-file-creation.sh # FIXME
+          sed 's/"stdx-allocator": "2.77.0",/"stdx-allocator": "2.77.2",/' -i dub.selections.json # upgrade stdx-allocator (can be removed once v1.11 gets released)
+          rm test/ddox.sh # can be removed once v1.11 gets released
+          sed -i '/^source.*activate/d' travis-ci.sh
+EOF
+        echo '      DC=\$DC ./travis-ci.sh'
+        ;;
+
+    dlang/tools)
+        # explicit test to avoid Digger setup, see dlang/tools#298 and dlang/tools#301
+        echo "      make -f posix.mak all DMD='dmd'"
+        echo "      make -f posix.mak test DMD='dmd' DFLAGS="
+        ;;
+
+    msgpack/msgpack-d)
+        echo '      DMD=\$DMD MODEL=64 make -f posix.mak unittest'
+        ;;
+
+    economicmodeling/containers)
+        echo '      make -B -C test/ || echo failed' // FIXME
+        ;;
+
+    BlackEdder/ggplotd)
+        # workaround https://github.com/BlackEdder/ggplotd/issues/34
+        echo "      sed -i 's|auto seed = unpredictableSeed|auto seed = 54321|' source/ggplotd/example.d"
+        use_travis_test_script
+        ;;
+
+    BBasile/iz)
+        echo '      cd scripts && sh ./test.sh'
+        ;;
+
+    dlang-community/D-YAML)
+        echo '      dub build --compiler=\$DC'
+        echo '      dub test --compiler=\$DC'
+        ;;
+
+    sociomantic-tsunami/ocean)
+        cat << EOF
+        git submodule update --init
+        make d2conv V=1
+        make test V=1 DVER=2 F=production ALLOW_DEPRECATIONS=1
+EOF
+        ;;
+
+    eBay/tsv-utils-dlang)
+        echo '      make test DCOMPILER=\$DC'
+        ;;
+
+    dlang-tour/core)
+        echo '      git submodule update --init public/content/en'
+        echo '      dub test --compiler=\$DC'
+        ;;
+
+    CyberShadow/ae)
+        # remove failing extended attribute test
+        echo '      perl -0777 -pi -e "s/unittest[^{]*{[^{}]*xAttrs[^{}]*}//" sys/file.d'
```

Comment:
CC @CyberShadow. I'm not entirely sure why extended file attributes aren't supported under OpenVZ, but do you have a better/more generic idea than this patch?